src/main/java/Task.java

```diff
@@ -0,0 +1,112 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task (String description) {
```

Comment:
An idea is that you could make this into an abstract class and have the different methods for the different classes in their respective class files! It's just an alternative way to do it that you can consider.